mp/src/vgui2/vgui_controls/consoledialog.cpp

```diff
@@ -423,33 +423,36 @@ void CConsolePanel::RebuildCompletionList(const char *text)
 
 	bool bNormalBuild = true;
 
-	// if there is a space in the text, and the command isn't of the type to know how to autocomplet, then command completion is over
+	// if there is a space in the text, and the command isn't of the type to know how to autocomplete, then command completion is over
 	const char *space = strstr( text, " " );
 	if ( space )
 	{
 		ConCommand *pCommand = FindAutoCompleteCommmandFromPartial( text );
-		if ( !pCommand )
-			return;
```

Comment:
Let's keep the early return, it's much nicer code to read this way. This would make lines 430->447 not need to be changed, and pointlessly shifted right one tab.

Reply:
Can't have a return there, it would result in function early-exiting whenever there is a space and no complete command

Reply:
Oh, duh.